Code/Framework/AzToolsFramework/AzToolsFramework/Asset/AssetUtils.cpp

```diff
@@ -122,13 +122,12 @@ namespace AzToolsFramework::AssetUtils
         AZStd::vector<AZStd::string>& enabledPlatforms)
     {
         // note that the current host platform is enabled by default.
```

Comment:
Remove the comment that is describing the line that is being removed as well

Reply:
According to @lumberyard-employee-dm , the comment is still accurate:
The host platform is enabled by default, but it only gets disabled if there is an explicit `"<host-platform>" : "disabled"` key in any merged Setreg file (edited) 

We are just making it, so that it can now be disabled.
Previously it wasn't able to be disabled.